student.rb

```diff
@@ -3,17 +3,19 @@
 class Student < Person
   attr_reader :classroom
 
-  def initialize(age:, name: 'Unknown', parent_permission: true)
-    super(age: age, name: name, parent_permission: parent_permission)
-    @classroom = nil
+  # rubocop:disable Style/OptionalBooleanParameter
+  def initialize(age, classroom, name = 'Unknown', parent_permission = true)
+    super(age, name, parent_permission)
+    @classroom = classroom
   end
+  # rubocop:enable Style/OptionalBooleanParameter
```

Comment:
- [ ] you shouldn't disable Rubocop cause it helps you to write clean code so please make sure to remove it